ci/xbuild/wheel.py

```diff
@@ -416,11 +416,38 @@ def _get_python_tag(self):
         minor = sys.version_info.minor
         return "cp" + str(major) + str(minor)
 
+
     def _get_abi_tag(self):
-        soabi = sysconfig.get_config_var("SOABI")
-        parts = soabi.split("-")
-        assert parts[0] == "cpython"
-        return "cp" + parts[1]
+        """
+        Return the ABI tag if available, otherwise just emulate it.
+        Adopted from pypa/wheel, see https://github.com/pypa/wheel/blob/a51977075740fda01b2c0e983a79bfe753567219/src/wheel/bdist_wheel.py#L67
+        """
+
+        def get_flag(var, fallback):
+            val = sysconfig.get_config_var(var)
+            if val is None:
+                self.log.report_abi_variable_missing(var)
+                return fallback
+            return val
+
+        soabi = sysconfig.get_config_var('SOABI')
+        if soabi:
+            parts = soabi.split("-")
+            assert parts[0] == "cpython"
+            abi = 'cp' + parts[1]
+        elif hasattr(sys, 'maxunicode'):
```

Comment:
According to https://docs.python.org/3/library/sys.html, `sys.maxunicode` was changed in 3.3, which means it existed even before that. Since we're not targeting pythons before 3.5, this branch should never happen. So I'd suggest changing this line to simple `else:`

Reply:
Yeah, I've cleaned up most of the old branches and this one could also be cleaned up. This PR is still WIP as I wanted to make sure wheels can be generated on all the platforms first. We can now refactor and adjust it.